kite-data/kite-data-core/src/test/java/org/kitesdk/data/spi/TestRefinableViews.java

```diff
@@ -819,4 +820,27 @@ public void testNotPartitioned() throws Exception {
     Assert.assertNotNull("with should succeed",
         notPartitioned.with("timestamp", now));
   }
+
+  @Test
+  public void testReadySignals() {
```

Comment:
I'd like to see a more exhaustive set of views tested by this method. Like the other tests in this file, this should make sure views backed by different constraints can be signaled correctly.